analyzer/src/funTest/assets/projects/external/directories-expected-output.yml

```diff
@@ -25,6 +25,7 @@ analyzer:
   config:
     ignore_tool_versions: false
     allow_dynamic_versions: false
+    sw360_configuration: null
```

Comment:
We do not want credentials to be serialized into the ORT result files. As this change adds the SW360 config to the ORT result it should also add `@JsonProperty(access = JsonProperty.Access.WRITE_ONLY)` to the `password` and `clientPassword` properties of `Sw360StorageConfiguration`, similar to `PostgresStorageConfiguration.password`. There should also be a test for this to prevent that it accidentally breaks and credentials are added to the ORT results. See e.g. `HttpFileStorageConfigurationTest`. I believe there was a similar test for `PostgresStorageConfiguration` but I couldn't find it anymore, maybe it was removed in a refactoring.